tests/Unit/Shared/Infrastructure/Fixture/Seeder/TestAuthorizationCodeManager.php

```
@@ -0,0 +1,41 @@
+
+final class TestAuthorizationCodeManager implements AuthorizationCodeManagerInterface
+{
+    private ?AuthorizationCodeInterface $savedCode = null;
+
+    public function __construct(
+        private readonly AuthorizationCodeInterface $existingCode
+    ) {
+    }
+
+    #[\Override]
+    public function find(string $identifier): ?AuthorizationCodeInterface
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$identifier'. (undefined)

(UnusedFormalParameter)